docs/koji.md

```diff
@@ -131,6 +131,10 @@ Example type-specific content generator metadata in context:
       "docker": {
         "id": "sha256:abc123def...",
         "parent_id": "sha256:123def456...",
+        "layer_sizes": [
+            {"sha256:123def456...": "1234556"},
+            {"sha256:456789013...": "1234"}
```

Comment:
If we make this a list and not a map, we'd better be clear in the documentation about which order it is.

Reply:
ahh right missed the description section,
yeah I wanted to keep the order

Reply:
added

Reply:
This has strings for the values, but aren't they really integers?

Reply:
yes will fix